packages/airview-compliance-ui/src/features/application-tile/application-tile-divider.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Divider from "@mui/material/Divider";
```

Comment:
For consistency, import as a named import: 

`import { Divider } from "@mui/material";`